src/components/Search/BookSearch.tsx

```
@@ -51,66 +65,63 @@ export default function BookSearch({SearchResultHeight, actions }: BookSearchPro
-          src={book.coverUrl}
-          alt={book.title}
-          className="w-full h-full object-cover"
-        />
-      </div>
+      style={{ msOverflowStyle: 'none', height: `calc(100vh - ${SearchResultHeight}px)`, scrollbarWidth: 'none' }}>
+                {Searchbooks.map((SearchBook) => (
+                  <div className="flex border-2 border-[var(--sub-color-2-brown,#EAE5E2)] rounded-2xl bg-[var(--White,#FFF)] shadow-sm"> 
+                  {/* 좌측 */}
+                    <div className="flex-1 flex p-[10px] gap-[20px]">
+                      {/* 썸네일 */}
+                      <div className="w-[136px] h-[192px] rounded-2xl overflow-hidden bg-gray-200 flex items-center justify-center">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)